### PR TITLE
[Refactor](Rf) refactor the code of runtime filter

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

File: be/src/runtime/fragment_mgr.cpp [hunk 1]
```diff
@@ -786,7 +786,7 @@ Status FragmentMgr::exec_plan_fragment(const TExecPlanFragmentParams& params,
     std::shared_ptr<RuntimeFilterMergeControllerEntity> handler;
     // TODO need check the status, but when I add return_if_error the P0 will not pass
     static_cast<void>(_runtimefilter_controller.add_entity(
```
Comment (BiteTheDDDDt) [inline be/src/runtime/fragment_mgr.cpp:788]: return if error?

Comment (yiguolei) [reply be/src/runtime/fragment_mgr.cpp:788]: zhiqiang hhh has tested, it will core .... but I donot know why

**Review by BiteTheDDDDt — DISMISSED** [review]

**Comment by HappenLee** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**